src/skins-qt/skin.cc

```diff
@@ -472,6 +473,23 @@ QFont * qfont_from_string (const char * name)
     QFont::Weight weight = QFont::Normal;
     QFont::Style style = QFont::StyleNormal;
     QFont::Stretch stretch = QFont::Unstretched;
+    static QHash<const QString, QFont::Weight> weightMap = {
+        {"Light", QFont::Light},
+        {"Regular", QFont::Normal},
+        {"Normal", QFont::Normal},
+        {"Roman", QFont::Normal},
+        {"Medium", QFont::Medium},
+        {"DemiBold", QFont::DemiBold},
+        {"Bold", QFont::Bold}
+    };
+    static QHash<const QString, QFont::Style> styleMap = {
+        {"Oblique", QFont::StyleOblique},
+        {"Italic", QFont::StyleItalic}
+    };
+    static QHash<const QString, QFont::Stretch> stretchMap = {
+        {"Condensed", QFont::Condensed},
+        {"Expanded", QFont::Expanded},
+    };
```

Comment:
I suspect that binary search (with a pre-sorted table created at compile-time) would be more efficient overall, taking into account CPU time + memory usage, than creating a hash at run-time.  tuple.cc has an example.

Reply:
Is that really a concern here, for a 2-element table that's created at startup?

Reply:
Probably not.  Your commit message says "More efficient lookups", so I was taking that at face value and wondering if it really is more efficient.  I would rather see a pull request to implement an actual font chooser in libaudqt, rather than making this stopgap code more efficient.